apps/web/pages/signup.tsx

```diff
@@ -360,13 +379,9 @@ export default function Signup({
                         router.push(`${GOOGLE_AUTH_URL}?${searchQueryParams.toString()}`);
                         return;
                       }
+                      setIsGoogleLoading(false);
```

Comment:
I don't think it is necessary to change the loading back to false,  if changing page with router.push is taking time, the loading spinner will stop before we are redirected